lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -0,0 +1,21 @@
+<div class="exam-timer">
+    <%- gettext("You are taking") %>
```

Comment:
Concatenation like this is not at all good for i18n. You need to pass the exam name as a named variable so translators can understand the complete phrase.